scribble-lib/scriblib/autobib.rkt

```diff
@@ -382,7 +384,15 @@
          (define (citet bib-entry . bib-entries)
            (add-inline-cite group (cons bib-entry bib-entries) the-style date<? date=?))
          (define (generate-bibliography #:tag [tag "doc-bibliography"] #:sec-title [sec-title "Bibliography"])
-           (gen-bib tag group sec-title the-style fn render-date-bib render-date-cite date<? date=? spaces))))]))
+           (gen-bib tag group sec-title the-style fn render-date-bib render-date-cite date<? date=? spaces))
+         #,(when (identifier? #'cite-author)
```

Comment:
An identifier can still not be provided and this form should still work. (For backwards compatibility.)

Although I suppose we can check to ensure that it is #f or an identifier and if not raise a syntax error.